contracts/test-helpers/IState_deployed.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.16;
+
+uint256 constant MAX_SMT_DEPTH = 64;
```

Comment:
The approach to move the contract here is not good enough.
What about it's dependencies: SMT, etc.
Where is the guarantee it is not changed by this time?
And what about the deploy scripts that were used? I see you've copied them but does not look scalable.
Let's discuss